tools/eslint/package.json

```diff
@@ -7,13 +7,12 @@
     "electron-to-chromium": "1.4.806"
   },
   "dependencies": {
-    "@babel/core": "^7.24.8",
```

Comment:
Why did you made these changes? Can you revert them, since they're completely unrelated to proposed changes.